recipes/cheat/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "cheat" %}
+{% set version = "2.1.26" %}
+{% set sha256 = "f5f6d86e7c1c3eb6c595156b534bc2a0305bd3a3433e20e0e78952a356aa347a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
+  skip: True  # [win and py3k]
```

Comment:
Due to a Windows/Python 3 [issue](aerkalov/ebooklib#104) with the ebooklib dependency, skip the Windows/Python 3 builds for now.